src/routes/authRouter.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { catchError } from '../utils/CatchErorr.js';
```

Comment:
There is a typo in the import path for `CatchErorr.js`. It should be `CatchError.js` to match the file name.